src/components/todo-filter.jsx

```diff
@@ -1,13 +1,37 @@
-import FilterLink from 'components/filter-link';
+import FilterRadio from 'components/filter-radio';
 import {allTodos, completedTodos, activeTodos} from 'redux/modules/filter-todos';
 
 const TodoFilter = () => (
-  <div>
-    Filter:
-    <FilterLink action={allTodos}>All</FilterLink>
-    <FilterLink action={completedTodos}>Completed</FilterLink>
-    <FilterLink action={activeTodos}>Active</FilterLink>
-  </div>
+  <fieldset className="filters">
+    <legend className="filters__title">Show: </legend>
+    <FilterRadio action={allTodos}>All</FilterRadio>
```

Comment:
Do you think it makes sense to change the name of this component to something more generic? So that if we decide to use something different in the future we won't change this component on only the child.

Reply:
If you have any suggestions - please add.

Reply:
How about `FilterOption` it can be radio button, link, or whatever we decide later.

Reply:
Sound good